### PR TITLE
Update test Support_Exec to match OpenJ9 java.vendor string

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: Andrew Leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins test sanity plinux

File: test/Java8andUp/src/org/openj9/test/support/Support_Exec.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public static void checkStderr(Object[] execArgs) {
 			execArgs = new String[baseArgsArray.length + args.length];
 			System.arraycopy(baseArgsArray, 0, execArgs, 0,
 					baseArgsArray.length);
-		} else if (vendor.indexOf("IBM") != -1) {
+		} else if (vendor.indexOf("IBM") != -1 || vendor.indexOf("OpenJ9") != -1) {
 			String full = System.getProperty("java.fullversion");
 			logger.debug("***" + full + "***");
```
Comment (smlambert) [inline test/Java8andUp/src/org/openj9/test/support/Support_Exec.java]: would also change logger.debug to logger.info - as that information is useful to print to console

Comment (andrew-m-leonard) [reply test/Java8andUp/src/org/openj9/test/support/Support_Exec.java]: ok will add that

**Comment by pshipton** [comment]
jenkins test sanity plinux

**Comment by pshipton** [comment]
jenkins copyright check

**Comment by pshipton** [comment]
jenkins line endings check

**Comment by AdamBrousseau** [comment]
jenkins copyright check

**Comment by AdamBrousseau** [comment]
jenkins line endings check

**Comment by AdamBrousseau** [comment]
[Line Endings Check](https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-LineEndingsCheck/1044/) Passed  
Not sure why Github isn't displaying the build

**Pull request merged**